content/en/docs/ambient/architecture/_index.md

```diff
@@ -11,15 +11,22 @@ test: n/a
 
 ## Ambient APIs
 
+To enforce L7 policies, add the `istio.io/use-waypoint` label to your resource to use waypoint for the labeled resource.
```

Comment:
It still feels weird to immediately jump into `use-waypoint` when that won't work if you haven't done `dataplane-mode` yet.

1. Add to ambient dataplane
2. `use-waypoint` if you then want L7

Which in practice probably looks like

```
### Labels
#### Adding Workloads To Ambient Dataplane
#### Enforcing L7 Policies In Ambient

Reply:
This PR only moves content; there are still a lot of gaps that need to be filled.